includes/integration.drush.inc

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @file
+ * Contains Drush-related hook implementations.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function integration_drush_command() {
+  $items = array();
+  $items['integration-export-document'] = array(
+    'description' => 'Export a document.',
+    'aliases' => array('ied'),
+    'arguments' => array(
+      'backend' => 'The backend machine name.',
+      'producer' => 'The producer machine name.',
+    ),
+    'options' => array(
+      'entity-id' => array(
+        'description' => 'Optional entity ID.',
+        'example-value' => '123',
+      ),
+    ),
+    'examples' => array(
+      'drush integration-export-document filesystem articles --entity-id=123'
+        => 'Exports article 123 using the filesystem backend and the articles producer.',
+    ),
+  );
+  $items['integration-process-import'] = array(
+    'description' => 'Process a consumer import.',
+    'aliases' => array('ipi'),
+    'arguments' => array(
+      'consumer' => 'The consumer machine name.',
+    ),
+    'examples' => array(
+      'drush integration-process-import articles'
+        => 'Imports documents using the articles consumer.',
+    ),
+  );
+  return $items;
+}
+
+/**
+ * Callback for the integration-export-document command.
+ */
+function drush_integration_export_document($backend_name, $producer_name) {
+  if (!$backend_name || !$producer_name) {
+    return drush_log(dt('Please specify at least a backend and a producer as arguments.'), 'error');
+  }
+
+  $producer = \Drupal\integration_producer\ProducerFactory::getInstance($producer_name);
```

Comment:
This is only available if the `integration_producer` module is enabled so we should move this command in the `integration_producer` module.